src/components/SlideButton/SlideButton.js

```diff
@@ -0,0 +1,142 @@
+import { ChevronsRight } from '@lana/b2c-mapp-ui-assets';
+
+const components = {
+  ChevronsRight,
+};
+
+const props = {
+  initialInstructionText: {
+    type: String,
+    default: 'Desliza para confirmar',
+  },
+  completedText: {
+    type: String,
+    default: 'Confirmado',
+  },
+};
+
+const methods = {
+  startSlide(event) {
+    if (this.completed) { return; }
+    this.initialMouseX = this.getMouseXPosFromEvent(event);
+    this.endPoint = this.getEndingPoint();
+    this.calculateSliderInitialWidth();
+    this.calculateSlideButtonInitialPosition();
+    this.updateSlideButton(0);
+    this.updateSlider(0);
+    this.startDrag = true;
+    this.sliderClass = 'started';
```

Comment:
Do not know if it makes sense to create a variable to hold this class like `startedSliderClass` ... 🤔

Reply:
Good idea, however I would prefer even more if this `.js` file never cared about any class names at all, because that's what the markup is for. Instead: the `.js` file should have some data/computed property that tells whether the class is needed and then the `:class="{ someClassName: someCondition }" binding is used in the markup.